internal/configs/version1/nginx-plus.tmpl

```diff
@@ -67,7 +67,10 @@ http {
         default upgrade;
         ''      close;
     }
-
+    map $http_upgrade $vs_connection_upgrade {
```

Comment:
Please rename before merging `$vs_connection_upgrade` to `$vs_connection_header` here and in the other places. `$vs_connection_header` better reflects the meaning of that variable. Note that `$connection_upgrade ` from `map $http_upgrade $connection_upgrade` is fine.